<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA Deembed</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/icon_resource/RS.ico</normaloff>
    <normalon>:/new/icon_resource/RS.ico</normalon>:/new/icon_resource/RS.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="diagramLabel">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>30</y>
      <width>295</width>
      <height>287</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/diagrams/Graphics/CalPlanes_150.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="portsButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portsEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>404</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1-4</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="portsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>407</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ports:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outerCalEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>344</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>outer_probes.cal</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outerLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>347</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>15</red>
          <green>87</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>15</red>
          <green>87</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Outer Cal:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outerCalButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>339</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="innerCalEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>374</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>inner_probes.cal</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="innerLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>377</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inner Cal:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="innerCalButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="portOrderCheckbox">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>297</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse S2P Port Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>outerCalButton</tabstop>
  <tabstop>innerCalButton</tabstop>
  <tabstop>portsButton</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>portOrderCheckbox</tabstop>
  <tabstop>outerCalEdit</tabstop>
  <tabstop>innerCalEdit</tabstop>
  <tabstop>portsEdit</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
